### PR TITLE
Lesson 1.8

### PR DESCRIPTION
**Pull request by stephskas**
connect to Airtable API, fetch data, store in state and render Titles

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,36 +1,53 @@
...
+					Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
+				},
+			})
+				.then((result) => result.json())
+				.then((result) => setTodoList(result.records));
+			setIsLoading(false);
+		} catch (error) {
+			console.log(error.message);
+		}
+	}, []);
```
Comment (royemosby) [inline src/App.js:23]: Cool! Good use of try/catch. In a production environment, I often use [toast messages](https://uxplanet.org/toast-notification-or-dialog-box-ae32ad53106d) to let the user know (in user-friendly language of course!) that something went awry.

File: src/index.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
+const root = ReactDOM.createRoot(document.getElementById("root"));
+root.render(
+	<React.StrictMode>
+		<App />
+	</React.StrictMode>
 );
```
Comment (royemosby) [inline src/index.js:10]: Good updating this!

**Review by royemosby — APPROVED** [review]
Code works as expected and is organized well for easy reading. Thanks!

File: src/App.js (hunk 1 above)
Comment (stephskas) [reply src/App.js:23]: Oh thanks! I'm excited to try that out.

**Pull request merged**